crates/pathfinder/fixtures/sequencer/valid_full_contract.json

```diff
@@ -0,0 +1,2971 @@
+{
```

Comment:
Why add this vs. using the existing fixture?

Reply:
I'll remove this redundancy :+1: